data/virt_autotest/guest_params_xml_files/slem_5_5_64_kvm_hvm_x86_64.xml

```diff
@@ -84,8 +84,7 @@
     <guest_autoconsole>text</guest_autoconsole>
     <guest_noreboot></guest_noreboot>
     <guest_default_target>graphical</guest_default_target>
-    <!-- Registration with scc.suse.com can only be enabled later until CDN publishment ready (bsc#1213947) -->
-    <guest_do_registration>false</guest_do_registration>
+    <guest_do_registration>true</guest_do_registration>
```

Comment:
As confirmed earlier, for development product test, please do registration with PROXY SCC. Official scc only works as expected AFTER milestone is announced. Anything more needed in the PR to let it work with proxy scc?

Reply:
It works with official scc at the moment. Why have to use proxy scc ?

Reply:
See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17740#issuecomment-1717047325.